examples/basic/src/main/java/com/soundcloud/lightcycle/sample/basic/BaseActivity.java

```diff
@@ -0,0 +1,12 @@
+package com.soundcloud.lightcycle.sample.basic;
+
+import android.support.v7.app.AppCompatActivity;
+
+import com.soundcloud.lightcycle.ActivityLightCycle;
+import com.soundcloud.lightcycle.LightCycleBaseClass;
+import com.soundcloud.lightcycle.LightCycleDispatcher;
+
+@LightCycleBaseClass
+public abstract class BaseActivity extends AppCompatActivity
+        implements LightCycleDispatcher<ActivityLightCycle<AppCompatActivity>> {
```

Comment:
Actually, looking more at the tests, the answer seems to be yes : 

```
@LightCycleBaseClass
public abstract class BaseActivityAsObject<T extends String> extends Activity
        implements LightCycleDispatcher<ActivityLightCycle<BaseActivityAsObject<T>>> {
}
```

do you confirm ?